eng/liveBuilds.targets

```diff
@@ -0,0 +1,95 @@
+<Project>
+
+  <PropertyGroup>
+    <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</HostArch>
```

Comment:
This is also called `BuildArchitecture`, why can't that be shared more broadly, or have one use the value of the other?

https://github.com/dotnet/runtime/blob/d2685d5c237ba0e4d16f94b3e918c627c5f26dbd/src/installer/Directory.Build.props#L48

`ToLowerInvariant` is also missing `()` in this PR's line, but I suppose I don't know if that's actually necessary in MSBuild.